src/code/z_rcp.c

```diff
@@ -821,7 +895,10 @@ Gfx* Gfx_SetFog2(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 n, s32 f) {
 }
 
 Gfx* Gfx_CallSetupDLImpl(Gfx* gfx, u32 i) {
-    gSPDisplayList(gfx++, &sSetupDL[6 * i]);
+    s32 dListIndex = 6 * i;
+
+    gSPDisplayList(gfx++, &((Gfx*)sSetupDL)[dListIndex]);
```

Comment:
i still feel like doing stuff like this is forcing it to be something that its not. but i guess we've always taken our liberties with changing the structure of something to make it better if it doesnt effect matching. 
just still dont feel 100% about it in this case

Reply:
I might be forcing it to be something it wasn't, but I'm not forcing it to be something it's not. It *is* a n x 6 array of Gfx commands. That Nintendo chose a strange way to handle it shouldn't make a difference.

Reply:
fair enough, we're looking to do the same thing with vrbox anyway. Still dont feel 100% about it, but will go with the majority here.
Which btw if you want to make the changes to the skybox palletes, is prob the only thing holding that up